src/main/webapp/app/overview/course-overview.component.scss

```diff
@@ -307,3 +307,18 @@ jhi-secured-image {
 .dropdown-toggle::after {
     display: none;
 }
+
+@media print {
+    .sidebar-container {
+        width: $menu-width-closed !important;
+    }
+
+    .mat-drawer {
+        width: $menu-width-closed !important;
+    }
+
+    .mat-drawer-content {
+        width: calc(100vw - $menu-width-closed) !important;
+        margin-left: $menu-width-closed !important;
+    }
+}
```

Comment:
I think that we should also completely hide this component instead.

If we "only" want to collapse it, then I think that it would be a better approach to set `isNavbarCollapsed` to true, and reset to the previous state after the printing is done.

Reply:
completely hiding it does not give us any benefit as the exam can not be moved to the free space (out of its own component) therefore only collapsing it should be the approach we take here 👍